src/briefcase/platforms/macOS/__init__.py

```diff
@@ -149,6 +149,14 @@ def _install_app_requirements(
                     "--platform",
                     f"macosx_{macOS_min_tag}_{self.tools.host_arch}",
                 ],
+                install_hint=f"""
+If an {self.tools.host_arch} wheel has not been published for one or more of your requirements,
```

Comment:
Really lucky that both arm and x86 start with a voiced vowel, so we can use "an" in both cases... :-)

With the removal of the newline from the exception usage, this will need an additional newline.
```suggestion
                install_hint=f"""
                
If an {self.tools.host_arch} wheel has not been published for one or more of your requirements,
```